db/schema.rb

```diff
@@ -117,6 +117,8 @@
     t.string "uid", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "terms_and_conditions_version_accepted"
+    t.datetime "terms_and_conditions_timestamp"
```

Comment:
Rails conventions are to use an `_at` suffix for timestamp fields. How about applying that here?

eg.

```suggestion
    t.datetime "terms_and_conditions_accepted_at"
```